package.json

```diff
@@ -130,7 +130,7 @@
     "express": "^4.15.4",
     "fun-hooks": "^0.9.9",
     "just-clone": "^1.0.2",
-    "live-connect-js": "2.4.0"
+    "live-connect-js": "3.0.0"
```

Comment:
Before merging we should wait for this fix https://github.com/LiveIntent/live-connect/pull/87 to be merged an use the new lc version